test/junit/scala/collection/immutable/RangeTest.scala

```diff
@@ -0,0 +1,30 @@
+package scala.collection.immutable
+
+import org.junit.{Assert, Test}
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import scala.tools.testing.AssertUtil
+
+@RunWith(classOf[JUnit4])
+class RangeTest {
+  import Assert._
+  import AssertUtil._
+
+  @Test
+  def test_SI10060_long_max(): Unit = {
+    assertEquals(Range.Int.inclusive(1, 9, 1).min, 1)
+    assertEquals(Range.Int.inclusive(1, 9, 1).max, 9)
```

Comment:
Also these tests should probably use `Range.Long`, since that's where the bug was and for consistency with `long` in the method name `test_SI10060_long_max`.